sdk/typescript/src/types/sui-bcs.ts

```diff
@@ -447,110 +328,152 @@ const BCS_SPEC = {
+      amount: 'Option<u64>',
+    },
+    PublishTx: {
+      modules: 'vector<vector<u8>>',
+    },
+    SharedObjectRef: {
+      objectId: BCS.ADDRESS,
+      initialSharedVersion: BCS.U64,
+      mutable: BCS.BOOL,
+    },
+    StructTag: {
+      address: BCS.ADDRESS,
+      module: BCS.STRING,
+      name: BCS.STRING,
+      typeParams: 'vector<TypeTag>',
+    },
+    MoveCallTx: {
+      package: BCS.ADDRESS,
+      module: BCS.STRING,
+      function: BCS.STRING,
+      typeArguments: 'vector<TypeTag>',
+      arguments: 'vector<CallArg>',
+    },
+    TransactionData: {
       kind: 'TransactionKind',
-      sender: 'address',
+      sender: BCS.ADDRESS,
       gasData: 'GasData',
     },
-  },
-
-  GasData: {
-    struct: {
+    GasData: {
       payment: 'SuiObjectRef',
-      owner: 'address',
-      price: 'u64',
-      budget: 'u64',
+      owner: BCS.ADDRESS,
+      price: BCS.U64,
+      budget: BCS.U64,
     },
-  },
-
-  // Signed transaction data needed to generate transaction digest.
-  SenderSignedData: {
-    struct: {
+    // Signed transaction data needed to generate transaction digest.
+    SenderSignedData: {
       data: 'TransactionData',
       txSignatures: 'vector<vector<u8>>',
     },
   },
+  aliases: {
+    ObjectDigest: BCS.BASE64,
+  },
 };
 
 const BCS_0_23_SPEC = {
-  ...BCS_SPEC,
-  MoveCallTx: {
-    struct: {
+  structs: {
+    ...BCS_SPEC.structs,
+    MoveCallTx: {
       package: 'SuiObjectRef',
-      module: 'string',
-      function: 'string',
+      module: BCS.STRING,
+      function: BCS.STRING,
       typeArguments: 'vector<TypeTag>',
       arguments: 'vector<CallArg>',
     },
-  },
-  SharedObjectRef: {
-    struct: {
-      objectId: 'address',
-      initialSharedVersion: 'u64',
+    SharedObjectRef: {
+      objectId: BCS.ADDRESS,
+      initialSharedVersion: BCS.U64,
     },
   },
+  enums: BCS_SPEC.enums,
+  aliases: {
+    ObjectDigest: BCS.BASE64,
+  },
 };
 
 const BCS_0_24_SPEC = {
-  ...BCS_SPEC,
-  SharedObjectRef: {
-    struct: {
-      objectId: 'address',
-      initialSharedVersion: 'u64',
+  structs: {
+    ...BCS_SPEC.structs,
+    SharedObjectRef: {
+      objectId: BCS.ADDRESS,
+      initialSharedVersion: BCS.U64,
     },
   },
+  enums: BCS_SPEC.enums,
+  aliases: {
+    ObjectDigest: BCS.BASE64,
+  },
 };
 
 // for version <= 0.26.0
 const BCS_0_26_SPEC = {
-  ...BCS_SPEC,
-  TransactionData: {
-    struct: {
+  structs: {
+    ...BCS_SPEC.structs,
+    TransactionData: {
       kind: 'TransactionKind',
-      sender: 'address',
+      sender: BCS.ADDRESS,
       gasPayment: 'SuiObjectRef',
-      gasPrice: 'u64',
-      gasBudget: 'u64',
+      gasPrice: BCS.U64,
+      gasBudget: BCS.U64,
     },
-  },
-  SenderSignedData: {
-    struct: {
+    SenderSignedData: {
       data: 'TransactionData',
       txSignature: 'vector<u8>',
     },
   },
+  enums: BCS_SPEC.enums,
+  aliases: {
+    // change in object digest to base58
+    ObjectDigest: BCS.BASE58,
```

Comment:
This should still be `BCS.BASE64` as 0.26.0 is still using base64. This might be why the CI is failing

Reply:
Good catch. Patched and waiting for CIs to finish.